scale-tests/configs/2018-08-08-mvrandom110s-full-scale.env

```diff
@@ -0,0 +1,111 @@
+# Depends on:
+# - TEST_NAME
+# - TEST_S3_BUCKET
+# - TEST_S3_FOLDER
+
+# Infrastructure configuration #################################################
+
+SERVICE_NAMES_PREFIX="${TEST_NAME}/"
+INFRASTRUCTURE_OUTPUT_FILE="${TEST_NAME}-infrastructure.json"
+KAFKA_ZOOKEEPER_CONFIG='scale-tests/configs/kafka-zookeeper-options.json'
+KAFKA_CLUSTER_COUNT=1
```

Comment:
Just curious, for testing DRF everything should be remain how we've done it in the past but based on Chris' comment about testing 1.12, he said we can change anything.
> this test can have as many variables changing as we can squeeze in) [2, 3, 4]. 

do we want to increase the count for the 2nd test? (if we get to it)

Reply:
We can always adjust things after the fact, but that would be a separate file -- or some set of steps that "scales up" the workload.